### PR TITLE
Make dispatch solo compact with funky knative

### PR DESCRIPTION
**Pull request by pzmrzy**

### CODE REVIEW

File: pkg/functions/docker/driver.go [hunk 1]
```diff
@@ -248,7 +248,7 @@ func (d *Driver) findActiveContainer(ctx context.Context, functionID, functionRe
 // GetRunnable creates runnable representation of the function
 func (d *Driver) GetRunnable(e *functions.FunctionExecution) functions.Runnable {
 	return func(ctx functions.Context, in interface{}) (interface{}, error) {
-		bytesIn, _ := json.Marshal(functions.Message{Context: ctx, Payload: in})
+		bytesIn, _ := json.Marshal(in)
```
Comment (berndtj) [inline pkg/functions/docker/driver.go]: I don't think it's this simple.  How are secrets passed?

Comment (pzmrzy) [reply pkg/functions/docker/driver.go]: Then how about modify funky, to remove non-necessary field before call python image?
@tenczar

Comment (tenczar) [reply pkg/functions/docker/driver.go]: For Knative Funky was modified to read the secrets from environment variables. This means that a function gets secrets passed to it at creation time. When a function is invoked, funky reads the secrets from the environment variables and adds them to the context that is passed to the language server.

**Comment by tenczar** [comment]
Berndt is correct that this PR does not completely handle all of the changes necessary to make Dispatch Solo work with the Knative Funky. Function creation needs to be modified to set the secrets as environment variables. These variables need to take the form of `SECRETS=secret_name_1, secret_name_2, etc`. Then each secret value should be another variable labeled `d_secret_{secret name}`, e.g `d_secret_secret_name_1`, `d_secret_secret_name_2`

**Comment by berndtj** [comment]
Also, this would be a GREAT opportunity to add e2e tests for solo (hopefully unmodified from the current e2e tests).

**Comment by pzmrzy** [comment]
Do you think we can remove all the code not related to dispatch-solo?

**Comment by berndtj** [comment]
I'm ok with cleaning up dead code for dispatch-solo, as long as we put it on a separate "solo" branch.

**Pull request closed without merging**